tests/model_registry/model_registry/upgrade/conftest.py

```
@@ -0,0 +1,120 @@
+) -> ModelRegistryClient:
+    server, port = model_registry_default_db_instance_rest_endpoint.split(":")
+    mr_client = ModelRegistryClient(
+        server_address=f"{Protocols.HTTPS}://{server}",
+        port=int(port),
+        author="opendatahub-test",
+        user_token=current_client_token,
+        is_secure=False,
+    )
+
+    if not mr_client:
+        raise ResourceNotFoundError("No model registry clients created")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)